twitter.css

```diff
@@ -0,0 +1,316 @@
+*{
+    box-sizing: border-box;
+}
+body{ margin: 0;
+}
+h2{
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+/* LAY OUT OF PAGE*/
+.main-grid {
+    display: grid;
+    grid-template: ". centered ." auto / 1fr auto 1fr;
+}
+.two-row-grid{
+    display: grid;
+    grid-template-rows: 50px auto;
+}
+.grid-auto__column {
+    display: grid;
+    grid-auto-flow: column;
+  
+    gap: 2px;
+  }
+.center-wrapper{
```

Comment:
Well done for using both methods for spacing. It gives you the chance to practise and use both methods.